packages/auth/src/Auth.ts

```diff
@@ -345,6 +346,7 @@ export class AuthClass {
 					);
 				});
 			}
+			autoSignIn = params['autoSignIn'] ? params['autoSignIn'] : false;
```

Comment:
Nit: for brevity, you could do:

```suggestion
			autoSignIn = params['autoSignIn'] || false;
```